### PR TITLE
Add functions

### PR DESCRIPTION
**Pull request by kapukap**

### CODE REVIEW

File: submissions/zahar/02-functions/index.js [hunk 1]
```diff
@@ -1,5 +1,71 @@
...
+//console.log(min(0,-10));
+
+
+
+// isEven(50) -> true
+// isEven(75) -> false
+// isEven(-1) -> false
+
+function isEven(numb){
+    if( numb%2 === 0){
```
Comment (zahar-g) [inline submissions/zahar/02-functions/index.js:19]: please fix spaces

Comment (kapukap) [reply submissions/zahar/02-functions/index.js:19]: Thanks!

File: submissions/kirill/02-functions/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+function countChar(str, char){
+    str = str.toLowerCase();
+    let count=0;
+    for(let i=0; i<str.length;i++){
+        if(str[i]===char){count++;}
+    }
+
+    return count;
+}
```
Comment (zahar-g) [inline submissions/kirill/02-functions/index.js:52]: don't forgot to add empty line at the end of each file.

Comment (zahar-g) [reply submissions/kirill/02-functions/index.js:52]: https://stackoverflow.com/a/2287990/3659811

File: submissions/zahar/02-functions/index.js [hunk 3]
```diff
@@ -1,5 +1,71 @@
...
+    let count=0;
+    for(let i=0; i<str.length;i++){
+        if(str[i]===char){count++;}
+    }
+
+    return count;
+}
+
+
+//console.log(countChar('Hello','l'));
```
Comment (zahar-g) [inline submissions/zahar/02-functions/index.js:71]: please revert this file back

**Pull request closed without merging**